src/index.html

```diff
@@ -11,6 +11,36 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <!-- <h1>Search bar airbnb</h1> -->
+
+    <form
+      action=" "
```

Comment:
set valid `action`

Reply:
Give me, please, a hint where I can find it, because I failed in it.)